main/com.spotify/eval/dev/annotated.txt

```diff
@@ -6428,11 +6428,20 @@ U: add this to playlist
 UT: $dialogue @org.thingpedia.dialogue.transaction.execute;
 UT: @org.thingpedia.media-source.get_currently_playing() => @com.spotify.add_song_to_playlist(song=id);
 ====
+U: add song to playlist
+UT: $dialogue @org.thingpedia.dialogue.transaction.execute;
+UT: @org.thingpedia.media-source.get_currently_playing() => @com.spotify.add_song_to_playlist(song=id);
+====
 # recording/61759f28c85549af58394e5794d978b8
 U: add this song to playlist named songs
 UT: $dialogue @org.thingpedia.dialogue.transaction.execute;
 UT: @org.thingpedia.media-source.get_currently_playing() => @com.spotify.add_song_to_playlist(playlist="songs", song=id);
 ====
+# recording/61759f28c85549af58394e5794d978b8
```

Comment:
Do not reuse IDs, and please give unique IDs to all data we add in annotated.txt, otherwise error analysis scripts will get confused.